chef/site-cookbooks/wca/recipes/default.rb

```diff
@@ -127,6 +127,17 @@
   options ['nodelaycompress','compress']
 end
 
+logrotate_app 'delayed_job-wca' do
+  path "#{repo_root}/WcaOnRails/log/delayed_job.log"
+  size "512M"
```

Comment:
I assume having two log files instead of one double how much disk space they'll use.

Could that be a problem?